docs/userguides/projects.md

```diff
@@ -47,6 +47,20 @@ a = accounts.load("metamask_0")
 a.deploy(project.MyContract)
 ```
 
+### Dependencies
+
+To set up dependencies in your ``ape-config.yaml`` file, follow [this guide](https://docs.apeworx.io/ape/stable/userguides/config.html#dependencies).
+If you are setting dependencies from a remote source, they will download when you run `ape compile` or other commands that compile in the background.
+Dependencies only need to download and compile once.
+
+You can access dependency contracts off your root project manager the same as your project's contracts.
+
+```python
+from ape import project
+
+dependency = project.MyDependency
```

Comment:
Yeah, I think if you had dependencies it wouldn't let you compile anything with the same name as something in your dependencies because of name shadowing, come to think of it.

Reply:
Correct, at this time (even before this PR), you would hit a naming collision if any contract type in your project manifest had the same name.